packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,409 @@
+                    />
+                    <Text align='center' size='xxs'>
+                        {localize('Scan the QR code to download Deriv X.')}
+                    </Text>
+                </>
+            );
+        } else if (acc_type === 'ctrader') {
+            return (
+                <>
+                    <QRCode
+                        value={mobileDownloadLink(acc_type, 'android')}
+                        size={5}
+                        style={{ height: 'auto', maxWidth: '100%', width: qr_code_mobile }}
+                    />
+                    <Text align='center' size='xxs'>
+                        {localize('Scan the QR code to download cTrader.')}
+                    </Text>
+                </>
+            );
+        } else if (acc_type === 'derivez') {
+            return (
+                <>
+                    <QRCode
+                        value={mobileDownloadLink(acc_type, 'android')}
+                        size={5}
+                        style={{ height: 'auto', maxWidth: '100%', width: qr_code_mobile }}
+                    />
+                    <Text align='center' size='xxs'>
+                        {localize('Scan the QR code to download Deriv EZ.')}
+                    </Text>
+                </>
+            );
+        }
+        return undefined;
+    };
+
+    const downloadCenterAppOption = (account_type: TPlatformType) => {
+        if (account_type === 'dxtrade') {
+            return (
+                <div className='cfd-trade-modal__download-center-app--option'>
+                    <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
+                        {localize('Run Deriv X on your browser')}
+                    </Text>
+                    <DxtradeDesktopDownload is_demo={is_demo} dxtrade_tokens={dxtrade_tokens} />
+                </div>
+            );
+        } else if (account_type === 'ctrader') {
+            return (
+                <>
+                    <div className='cfd-trade-modal__download-center-app--option'>
+                        <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
+                            {localize('Run cTrader on your browser')}
+                        </Text>
+                        <CTraderDesktopDownload />
+                    </div>
+                    <div className='cfd-trade-modal__download-center-app--option'>
+                        <Icon icon='IcWindowsLogo' size={32} />
+                        <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
+                            {localize('cTrader Windows app')}
+                        </Text>
+                        <a
+                            className='dc-btn cfd-trade-modal__download-center-app--option-link'
+                            type='button'
+                            href={'https://ctrader.com/download/'}
```

Comment:
Better not to hard code this here.
Define a function in `logic.ts` same as `getPlatformMt5DownloadLink` and move the links there.